### PR TITLE
Prioritize Development Token Provided in TP_DEV_TOKEN variable over token in constructor

### PR DESCRIPTION
**Pull request by Artem-Kuz**

### CODE REVIEW

**Comment by s-glatshtein** [comment]
@eldarkra looks good. can you approve please?

**Review by mstrelex — APPROVED** [review]

**Comment by mstrelex** [comment]
@s-glatshtein Approved, please rebase and merge.

File: TestProject.OpenSDK/Internal/Rest/AgentClient.cs [hunk 1]
```diff
@@ -228,13 +228,18 @@ private AgentClient(Uri remoteAddress, string token, DriverOptions capabilities,
             ReportSettings sessionReportSettings = disableReports ? null : this.InferReportSettings(reportSettings);
             this.reportSettings = sessionReportSettings;
 
-            if (token != null)
+            if (Environment.GetEnvironmentVariable(this.tpDevToken) != null)
             {
-                this.token = token;
+                if (token != null)
+                {
+                    Logger.Info("Using {0} parameter from environment", this.tpDevToken);
```
Comment (vitalybu) [inline TestProject.OpenSDK/Internal/Rest/AgentClient.cs:235]: Sensitive data such as a token should not be printed as INFO... Maybe DEBUG.

**Review by vitalybu — CHANGES_REQUESTED** [review]

File: TestProject.OpenSDK/Internal/Rest/AgentClient.cs (hunk 1 above)
Comment (s-glatshtein) [reply TestProject.OpenSDK/Internal/Rest/AgentClient.cs:235]: tpDevToken is the field containing the name of the environment variable.
We should change it to a real constant at some point....

**Review by vitalybu — APPROVED** [review]

**Pull request merged**